walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,111 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Collection;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
 
+/** Find available meeting time slots */
 public final class FindMeetingQuery {
+  /** Get events of requested attendees */
+  private List<Event> getRelevantEvents(Collection<Event> events, Collection<String> requestedAttendee){
+    List<Event> relevantEvents = new ArrayList<>();
+    Iterator<Event> iter = events.iterator();
+    Event currEvent;
+    Set<String> meetingAttendee = new HashSet<>();
+    while(iter.hasNext()){
+      currEvent = iter.next();
+      meetingAttendee.addAll(currEvent.getAttendees());
```

Comment:
After moving meetingAttendee into the for loop, you can instantiate the set by passing in a collection. [Source](https://docs.oracle.com/javase/8/docs/api/java/util/HashSet.html)